satpy/scene.py

```diff
@@ -78,10 +78,10 @@ def __init__(self, filenames=None, reader=None, filter_parameters=None, reader_k
         To load data `filenames` and preferably `reader` must be specified. If `filenames` is provided without `reader`
         then the available readers will be searched for a Reader that can support the provided files. This can take
         a considerable amount of time so it is recommended that `reader` always be provided. Note without `filenames`
-        the Scene is created with no Readers available requiring Datasets to be added manually::
+        the Scene is created with no Readers available requiring DataArrays to be added manually::
 
             scn = Scene()
-            scn['my_dataset'] = Dataset(my_data_array, **my_info)
+            scn['my_dataset'] = xr.DataArray(my_data_array, **my_info)
```

Comment:
```suggestion
            scn['my_dataset'] = xr.DataArray(my_data_array, attrs=my_metadata)
```
`my_info` is supposed to be metadata.